### ISSUE TITLE
More information about training

### ISSUE THREAD

**Issue by BlingHe** [csyxwei/ELITE#4]
Hi, 

Thanks for sharing this interesting work!
Would you provide more information about training the Global Mapping Network and Local Mapping Network. For example, Which type of GPU you used? How long does each model take to be trained?

Thank you very much for your help, and I look forward to hearing back from you soon.

Best regards, Bling.

**Comment by csyxwei** [comment]
Hi, 

We train our global mapping network and local mapping network on 4 V100 GPUs. The hyperparameters we used are provided in `train_global.sh` and `train_local.sh`.

The global mapping network is trained for 7w iterations, and the local mapping network is trained for 6w5 iterations. The total training time is about one week.

**Comment by BlingHe** [comment]
> Hi,
> 
> We train our global mapping network and local mapping network on 4 V100 GPUs. The hyperparameters we used are provided in `train_global.sh` and `train_local.sh`.
> 
> The global mapping network is trained for 7w iterations, and the local mapping network is trained for 6w5 iterations. The total training time is about one week.

Thanks for your reply, that helps me a lot.  And i still have some confusions. 
Why did you choose 7w or 6w5 as the total steps? As far as I know, the LDM loss will fluctuate throughout the training process and is there an indicator to guide us to determine whether the model converges?

**Comment by csyxwei** [comment]
Hi, sorry for the late reply.  The total number of training steps we use is an empirical value, and we find that these steps are sufficient for training the ELITE.
